### DISCUSSION TITLE
Welcome to Ice Cream Machine Core Discussions!

### DISCUSSION THREAD

**Discussion by rockerbacon** [rockerbacon/ice-cream-machine-core#2] [Announcements]
## Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together.

**Comment by rockerbacon** [comment]
Hello, we're just discussing some stuff to see how it works. @nlaranjeira01 say something funny

**Reply by nlaranjeira01** [reply]
Despretensioso

**Comment by nlaranjeira01** [comment]
Comment
